### PR TITLE
Present certain TOC rendering options to the user through Atom config

### PR DESCRIPTION
**Pull request by tanhevg**
- [x] I have looked over the [contribution guide](https://github.com/atom-community/markdown-preview-plus/blob/master/CONTRIBUTING.md), which contains some crucial technical information.
#468

### CODE REVIEW

File: src/markdown-it-helper.ts [hunk 1]
```diff
@@ -69,7 +71,14 @@ function init(initState: InitState): markdownItModule {
   if (initState.checkBoxes) markdownIt.use(require('markdown-it-task-lists'))
   if (initState.toc) {
     markdownIt.use(require('markdown-it-anchor'))
-    markdownIt.use(require('markdown-it-table-of-contents'))
+    const _includeLevels = []
+    for (let _i = 1; _i <= initState.tocDepth; _i++) {
+      _includeLevels.push(_i)
+    }
```
Comment (lierdakil) [inline src/markdown-it-helper.ts]: This looks somewhat ugly. Can't we shorten it? F.ex.
```js
// snip
// array [1, 2, …, initState.tocDepth]
includeLevel: Array.from({length: initState.tocDepth}, (x, i) => i+1)
// snip
```

**Comment by lierdakil** [comment]
Thanks for submitting a pull request! I have a minor nitpick about the code style, it'd be great if you could fix that. If you don't have time to do that, no worries, I can do that myself, but that'll probably take a few days, since I'm short on spare time myself at the moment.

**Comment by tanhevg** [comment]
Fixed. No worries, JavaScript/TypeScript is not my first language atm, so any style tips are appreciated.

**Comment by lierdakil** [comment]
Thanks! Released as v3.11.0

**Pull request merged**